src/FrontEnd/Pages/ParticipantSide/01_activity_screen.tsx

```diff
@@ -1,139 +1,317 @@
-
-                                </div>
-
-                                <div className="option">
-
-                                </div>
-
-                            </div>
-
-                            <div className="question-box">
-
-                                <div className="question">
-
-                                </div>
-
-                                <div className="option">
-
-                                </div>
-
-                            </div>
-
-                            <div className="question-box">
-
-                                <div className="question">
-
-                                </div>
-
-                                <div className="option">
-
-                                </div>
-
-                            </div>
-                        </div>
-
-
-
-                    </div>
-
-                </section>
-            </React.Fragment>
-        );
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import { useNavigate } from "react-router-dom";
+import Draggable from "react-draggable";
+import { useLocation } from "react-router-dom";
+
+const API = axios.create({
+  baseURL: "https://cs565-backend-2023.herokuapp.com/",
+});
+
+type Symbol = {
+  name: string;
+  pitch?: number;
+};
+
+type MeasureResponse = {
+  userId: string;
+  symbols: Symbol[];
+};
+
+type RequestBody = {
+  measureId: string;
+  measureResponse: MeasureResponse;
+};
+
+type Note = {
+  type: number;
+  x: number;
+  y: number;
+};
+
+const types = [0, 1, 2, 3, 4, 5];
+const typesName = [
+  "whole_note",
+  "half_note",
+  "quarter_note",
+  "whole_rest",
+  "half_rest",
+  "quarter_rest",
+];
+
+const notePos = [375, 350, 325, 300, 275, 250, 225, 200, 175];
+const pitchIdx = new Map(notePos.map((val, idx) => [val, idx]));
+
+const ParticipantActivityScreen = () => {
+  const [measure, setMeasure] = useState({ _id: "", image: "" });
+  const [num_submission, setNum_submission] = useState(0);
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    fetchMeasure();
+  }, [num_submission]);
+
+  const fetchMeasure = () => {
+    API.get("api/measuretask")
+      .then((response) => {
+        const data = response.data;
+        let u8 = new Uint8Array(data.data.image.data);
+        let decoder = new TextDecoder("utf8");
+        let b64 = decoder.decode(u8);
+        setMeasure({
+          _id: data.data._id,
+          image: `data:image/png;base64,${b64}`,
+        });
+      })
+      .catch((error) => {
+        console.error("There was a problem with the fetch operation:", error);
+      });
+  };
+
+  const handleSubmitClick = () => {
+    setNum_submission((prevState) => prevState + 1);
+    const symbols: Symbol[] = notes.map((note: Note) => {
+      if (note.type === 0 || note.type === 1 || note.type === 2) {
+        return {
+          name: String(typesName[note.type]),
+          pitch: pitchIdx.get(note.y),
```

Comment:
- [ ] goldSymbols assumes that the bottom most line is pitch "1", so change to 1-indexed instead of 0-indexed